googletrans/client.py

```diff
@@ -35,8 +35,8 @@ def __init__(self, service_urls=None, user_agent=DEFAULT_USER_AGENT):
         self.session.headers.update({
             'User-Agent': user_agent,
         })
-        self.token_acquirer = TokenAcquirer(session=self.session)
         self.service_urls = service_urls or ['translate.google.com']
+        self.token_acquirer = TokenAcquirer(session=self.session, host=service_urls[0])
```

Comment:
This line makes [test fails](https://travis-ci.org/ssut/py-googletrans/jobs/220487929#L419), since the `service_urls` above is binded to the class attribute `self.service_urls` and covers the case if `service_urls` is not set. so the param should be `self.service_urls[0]` instead.